src/store/normalize-data.js

```diff
@@ -122,10 +121,9 @@ const addNode = state => node => {
   state.node.tags[id] = node.tags || [];
   // supports for metadata in case it exists on initial load
   state.node.code[id] = node.code;
-  state.node.codeLocation[id] = node.codeLocation;
   state.node.docString[id] = node.docString;
   state.node.parameters[id] = node.parameters;
-  state.node.dataset_location[id] = node.dataset_location;
+  state.node.filepath[id] = node.filepath;
   state.node.dataset_type[id] = node.dataset_type;
```

Comment:
I didn't pick up on this last time, but it'd be great for us to be consistent in our filename schema when adding things to the store, i.e. consistently using camelCase over underscores. This need not apply to the API endpoint itself necessarily, but it'd be great to align the variable names we're using in the main store.
```suggestion
  state.node.filepath[id] = node.filepath;
  state.node.datasetType[id] = node.dataset_type;
```
However I note that the API appears to be using both `docString` and `dataset_type` which isn't ideal

Reply:
yea good call about this, I also thought about changing this earlier too - I'll make the change 👍